### DISCUSSION TITLE
Reset Tesla Token

### DISCUSSION THREAD

**Discussion by DomiSanNo1** [pkuehnel/TeslaSolarCharger#953] [Q&A]
Hey guys, is it possible to reset the token and the refresh token without a new pull or installation? I only need to save the new tokens.
Thank you!

**Comment by pkuehnel** [comment]
If you change the Encryption key in your docker-compose yml (teslaMate and TEslaMate API) and use `docker compose up -d` after that, TeslaMAte can't read the Token and should force you to enter a new Token.
